### DISCUSSION TITLE
Real Car Tutorial Feedback thread

### DISCUSSION THREAD

**Discussion by schmittlema** [prl-mushr/mushr#88] [Software]
This thread is for feedback and questions on the real car tutorial/setup

**Comment by Lilly-yang** [comment]
Hi Matt,

Could you please tell me how to upgrade Jetson Nano to Ubuntu 20.04 safely? Because JetPack 4.6 is the last major release for Jetson Nano and it provides Ubuntu 18.04.

Thank you!
Li

**Reply by schmittlema** [reply]
Hey Li
We have chosen to not upgrade to 20.04 and use docker instead. This is because [20.04 is not supported](https://forums.developer.nvidia.com/t/ubuntu-20-04-on-nano/125451), and we would like to stick to supported software as much as possible. Here is a [tutorial](https://qengineering.eu/install-ubuntu-20.04-on-jetson-nano.html) for upgrade you could try.
